docs/content/guide/component.ngdoc

```diff
@@ -109,13 +109,16 @@ change will be reflected in the parent component. For components however, only t
 the data should modify it, to make it easy to reason about what data is changed, and when. For that reason,
 components should follow a few simple conventions:
 
-  - Inputs are realized with `@` and `=` bindings
+  - Inputs are realized with `<` bindings. The `<` symbol denotes {@link $compile#-scope- one-way bindings} which are
```

Comment:
You are right. For the type of things that can be passed with `@`, there shouldn't be any perf gain over `<` (except for some absolutely negligible initial setup (watching + immediately unwatching)).

So, I'm fine with replacing `@` with `<` (but one has to wrap the constant values in quotes).